modules/ballerina-http/src/main/java/org/ballerinalang/net/http/HttpUtil.java

```diff
@@ -711,4 +714,41 @@ public static void operationNotAllowedCheck(BStruct bStruct) {
         }
     }
 
+    /**
+     * Get multiparts.
+     *
+     * @param context                Context
+     * @param abstractNativeFunction AbstractNativeFunction
```

Comment:
Shall we have a meaningful description here rather than repeating the variable name("AbstractNativeFunction") itself?